src/vumps_mpo.jl

```diff
@@ -1,3 +1,9 @@
+function ITensors.NDTensors.contraction_output(
+  A::NDTensors.EmptyTensor, B::NDTensors.DiagBlockSparseTensor, label
+)
+  return ITensor(eltype(B), label)
+end
+
```

Comment:
I think there would be a better way to achieve this (at the very least, the `ITensor` type shouldn't be used in `NDTensors` code).

Reply:
Indeed, I replaced the output by NDTensors.EmptyTensor(eltype(B), label) which seems to work just as well.

I also moved it to ITensors.jl as it should be removed at some point.